queries.sql

```diff
@@ -8,4 +8,70 @@ SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
 SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = TRUE;
 SELECT * FROM animals WHERE name <> 'Gabumon';
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+BEGIN;
+
+UPDATE animals SET species = 'Unspecified';
+
+SELECT * FROM animals;
+
+ROLLBACK;
+
+SELECT * FROM animals;
+
+BEGIN ;
+
+UPDATE animals SET species = 'Digimon' WHERE name LIKE '%mon';
+
+UPDATE animals SET species = 'Pokemon' WHERE species IS NULL;
+
+SELECT * FROM animals;
+
+COMMIT;
+
+SELECT * FROM animals;
+
+BEGIN ;
+
+DELETE FROM animals;
+
+SELECT * FROM animals;
+
+ROLLBACK;
+
+SELECT * FROM animals;
+
+BEGIN ;
+
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+
+SAVEPOINT my_savepoint;
+
+UPDATE animals SET weight_kg = weight_kg * -1;
+
+ROLLBACK TO SAVEPOINT my_savepoint;
+
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
+
+COMMIT;
```

Comment:
- [ ] Your screenshot for these queries only had the queries but they didn't include the results so I am unable to confirm whether they are returning the right results or not so kindly fix that as well :+1: 
![image](https://github.com/waelelsafty07/vet/assets/102699958/02c0ab26-9574-436a-85d3-fe1fa03b46b5)